fdbserver/tester.actor.cpp

```diff
@@ -1017,11 +1017,40 @@ vector<TestSpec> readTests( ifstream& ifs ) {
 	return result;
 }
 
+ACTOR Future<Void> monitorServerDBInfo(Reference<AsyncVar<Optional<ClusterControllerFullInterface>>> ccInterface,
+                                       LocalityData locality,
```

Comment:
Is a non-empty locality ever passed in from the remaining callers?

Reply:
no, but there is a FIXME in the tester to add a locality, so I figured I would leave it as a parameter